docs/sources/developers/plugins/metadata.md

```diff
@@ -9,28 +9,128 @@ aliases = ["/docs/grafana/latest/plugins/developing/plugin.json/"]
+| `annotations`  | boolean                 | false    | For data source plugins. If the plugin supports annotation queries.                                                                  |
+| `backend`      | boolean                 | false    | If the plugin has a backend component.                                                                                               |
+| `category`     | string                  | false    | Plugin category used on the Add data source page. Possible values are: `tsdb`, `logging`, `cloud`, `tracing`, `sql`.                 |
+| `executable`   | string                  | false    | The first part of the file name of the backend component executables (there can three binaries for linux, darwin and windows).       |
+| `hasQueryHelp` | boolean                 | false    | For data source plugins. TODO not sure if this is used.                                                                              |
+| `logs`         | boolean                 | false    | For data source plugins. If the plugin supports logs.                                                                                |
+| `metrics`      | boolean                 | false    | For data source plugins. If the plugin supports metric queries. Used in the Explore feature.                                         |
+| `mixed`        | boolean                 | false    | Not to be used by external plugins. Special property for the built-in mixed plugin.                                                  |
+| `queryOptions` | [object](#queryoptions) | false    | For data source plugins. There is a query options section in the plugin's query editor and these options can be turned on if needed. |
+| `routes`       | array                   | false    | For data source plugins. Proxy routes used for plugin authentication and adding headers to HTTP requests made by the plugin.         |
+| `sort`         | number                  | false    | Internal property for sorting. Cannot be used as will be overwritten by Grafana.                                                     |
+| `streaming`    | boolean                 | false    | For data source plugins. If the plugin supports streaming.                                                                           |
+| `tracing`      | boolean                 | false    | For data source plugins. If the plugin supports streaming.                                                                           |
+
+## dependencies
+
+Plugin dependencies.
+
+### Properties
+
+| Property         | Type   | Required | Description                                                |
+|------------------|--------|----------|------------------------------------------------------------|
+| `grafanaVersion` | string | **true** | Required Grafana version for this plugin. TODO - format.   |
```

Comment:
I think @marefr can help here. Should be something something semver :-)

Reply:
The grafana.com have some logic for showing Grafana dependency and seems it works best if you use a format of `major.minor.path`, for example:
`Required Grafana version for this plugin. For example `6.x.x 7.x.x` if plugin requires Grafana v6.x.x and v7.x.x.`

For reference see Grafana dependency in right column for https://grafana.com/grafana/plugins/raintank-worldping-app

Reply:
Added a regex validator to versions, as well as examples in the description.